### ISSUE TITLE
Can't display font awesome icons in <v-text>

### ISSUE THREAD

**Issue by ashleymoogle** [konvajs/vue-konva#44]
Hi, vue-konva can't seem to be able to display font-awesome icons, it show a square-like 404 icon instead.
Here is a codepen with that behaviour in action: [https://codepen.io/ashyslashy/pen/WmoymO](https://codepen.io/ashyslashy/pen/WmoymO).
Am I doing something wrong? What can I do to make icons appear? Thanks.

**Comment by ashleymoogle** [comment]
Yeah the issue is also that font awesome v5 uses a main class called `fas` instead of `fa` so if the lib generates elements with the v4 syntax, it can't possibly work.
Also on kind of the same topic, inlining some props doesn't work at all, for example the font-family and font-size props, they have to be initialized in an object and bind that object to the component. Some other props are working fine.

**Comment by lavrton** [comment]
Can you give more info about the second issue?

**Comment by mtriff** [comment]
FYI for anyone trying to get FontAwesome v5 to work with `vue-konva`, you need to specify the following attributes on your `v-text` element:
```
            fontFamily: '"Font Awesome 5 Free"',
            fontStyle: 'bold'
```
The double quotes inside the `fontFamily` attribute are necessary.

**Comment by michaelwilson** [comment]
If you look [here](https://jsbin.com/jutitec/edit?html,js,output) you'll see that is displays a "hollow" star. 

The Font Awesome bundle specific is a "Pro" bundle, so what it _should_ display is a[Duotone star](https://fontawesome.com/icons/star?style=duotone).

Even if we're getting the "Free" FontAwesome, font weight (fontStyle) 900 should show a solid start, and 400 should show an outline star. If you change the 900 to 400, you'll see it doesn't change.

Here's the wacky part: in my code, when I ran Konva 6 from the CDN, and specified the Font Name as '"Font Awesome 5 Pro"'
I could get the font to change from outline to solid by using FontStyle, but still no duotone.

(Yes, I use FontFaceObserver to make sure the font is loaded).

Thanks!

**Comment by lavrton** [comment]
Can you add into the demo the `<i/>` that displays what you want (regular font awesome usage).

**Comment by michaelwilson** [comment]
Done! 

I just added these lines:


  Duotone: <i class="fad fa-star"></i>
  Light: <i class="fal fa-star"></i>
  Regular: <i class="far fa-star"></i>
  Solid: <i class="fas fa-star"></i>

**Comment by lavrton** [comment]
So first, the font family here is different. You need `"Font Awesome 5 Duotone"`.

The second,`Duotone` variant is complex. It consists of two shapes (grey and black). So it is not a single character! I don't think you can simply draw it with Konva.

**Comment by michaelwilson** [comment]
Ok I can live without the DuoTone. But can you explain why I can't get it to change to outline / solid by changing fontStyle here, or with Konva 7?

**Comment by lavrton** [comment]
Works just fine: https://jsbin.com/fitenemika/1/edit?html,js,output
